src/python/py_utils.cpp

```diff
@@ -179,11 +305,16 @@ ov::AnyMap kwargs_to_any_map(const py::kwargs& kwargs) {
     for (const auto& item : kwargs) {
         std::string key = py::cast<std::string>(item.first);
         py::object value = py::cast<py::object>(item.second);
-        if (utils::py_object_is_any_map(value)) {
+        // we need to unpack only dictionaries, which are passed with "config" name,
+        // because there are dictionary properties that should not be unpacked
+        if (utils::py_object_is_any_map(value) && key == "config") {
             auto map = utils::py_object_to_any_map(value);
             params.insert(map.begin(), map.end());
         } else {
-            params[key] = utils::py_object_to_any(value);
+            if (py::isinstance<py::none>(value)) {
+                continue;
```

Comment:
`generate("", eos_token_id=None)` probably means it should be set to `-1` instead of being ignored here. Maybe we should just report an error for `None`s?

Reply:
BTW, `py_object_to_any` can handle `py::none`. So, maybe we need to handle `None`s only there?

I'm Ok to throw exception, but alternatively we can allow `None` for properties with default meaningful ctors like `negative_prompt_3` (std::string) or `Adapters`, e.g.:
```python
    image = pipe.generate(prompt, adapters=None)
```
instead of:
```python
    image = pipe.generate(prompt, adapters=openvino_genai.AdapterConfig())
```
@slyalin do you have any preferences?

Reply:
`None`s as default values arise when there's a need to handle unspecified value alongside with the predefined default value in function args. Genai is in different situation. If something is unspecified we simply don't use that arg at all. So the example `image = pipe.generate(prompt, adapters=None)` should just be `image = pipe.generate(prompt)` in my understanding. Would that work?

Reply:
`image = pipe.generate(prompt, adapters=None)` implies "don't use adapters, which were set in Pipeline ctor". E.g. pipeline can be created with multiple LoRAs which are configurable in runtime (`generate(..)`) and `None` means don't it LoRA at all.

It's rather a syntax sugar, so probably throwing an exception is a simplier solution.

Reply:
Looks like throwing an exception in this case is a better solution because:
1. I agree that option=None is the closest to not mentioning option at all.
2. Having adapters=None to be an alternative to adapters=AdapterConfig() contradicts the previous point, and making options "do not change the adapters" and "set empty adapters" nearly undistinguishable because in the first case, we shouldn't mention adapters at all, and in the second case, we can pass adapters=None, that is hardly decodable from the user perspective.

Reply:
Added throwing of exception.